### PR TITLE
Add account role info modal

### PR DESCRIPTION
**Pull request by yuxitang-amzn**
*Issue #, if available:*

*Description of changes:*
Add account role info modal.

Diff from mockup:
* Confirmed with UX designer that backend role should be shown.

Mockup:
image

Implementation:
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/apps/account/role-info-panel.tsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  return (
+    <EuiOverlayMask>
+      <EuiModal onClose={props.handleClose}>
+        <EuiSpacer />
+        <EuiModalBody>
+          <EuiTitle>
+            <h4>Roles ({roles.length})</h4>
+          </EuiTitle>
+          <EuiText color="subdued">Roles you are currently mapped to by your administrator</EuiText>
```
Comment (tianleh) [inline public/apps/account/role-info-panel.tsx:56]: Based on my experience with UX, they will call out the missing period at the end of the sentence during Fit&Fitness session.

File: public/apps/account/role-info-panel.tsx [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+              {item}
+              <br />
+            </EuiText>
+          ))}
+          <EuiHorizontalRule />
+          <EuiTitle>
+            <h4>Backend roles ({backendRoles.length})</h4>
+          </EuiTitle>
+          <EuiText color="subdued">
+            Backend roles you are currently mapped to by your administrator
```
Comment (tianleh) [inline public/apps/account/role-info-panel.tsx:69]: Same here.

File: public/apps/account/role-info-panel.tsx (hunk 1 above)
Comment (yuxitang-amzn) [reply public/apps/account/role-info-panel.tsx]: Fixed.

File: public/apps/account/role-info-panel.tsx (hunk 2 above)
Comment (yuxitang-amzn) [reply public/apps/account/role-info-panel.tsx]: Fixed.

File: public/apps/account/role-info-panel.tsx [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+        <EuiModalBody>
+          <EuiTitle>
+            <h4>Roles ({roles.length})</h4>
+          </EuiTitle>
+          <EuiText color="subdued">
+            Roles you are currently mapped to by your administrator.
+          </EuiText>
+          <EuiSpacer />
+          {roles.map((item) => (
+            <EuiText>
```
Comment (tianleh) [inline public/apps/account/role-info-panel.tsx:61]: You may need a unique key for each of them to fix "Each child in a list should have a unique "key" prop."

File: public/apps/account/role-info-panel.tsx [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+          <EuiTitle>
+            <h4>External identities ({backendRoles.length})</h4>
+          </EuiTitle>
+          <EuiText color="subdued">
+            External identities you are currently mapped to by your administrator.
+          </EuiText>
+          <EuiSpacer />
+          <EuiListGroup>
+            {backendRoles.map((item) => (
+              <EuiText>
```
Comment (tianleh) [inline public/apps/account/role-info-panel.tsx:76]: same here

File: public/apps/account/role-info-panel.tsx [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+import {
+  EuiListGroup,
+  EuiModal,
+  EuiModalBody,
+  EuiOverlayMask,
+  EuiPanel,
```
Comment (tianleh) [inline public/apps/account/role-info-panel.tsx:20]: unused import

**Pull request merged**